app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,15 @@
 <% content_for :body do %>
   <div class="smart_answer">
+    <% if @presenter && @presenter.start_node.survey_url(request.path) %>
+      <div class="banner-container">
```

Comment:
We can update `<div class="banner-container">` to `<div class="banner-container govuk-!-static-margin-top-4">`.

This will apply 20px margin to the top of the div element and allow us to remove the `_intervention.scss` component stylesheet and import.


Further info - https://design-system.service.gov.uk/styles/spacing/

Reply:
Thanks, that's a great suggestion!